src/settings/profile.hpp

```diff
@@ -101,7 +101,41 @@ class Profile
 
         if (game == btu::Game::FNV)
         {
-            profile.bsa_make_overrides = true;
+            profile.bsa_allow_compression = false;
+            profile.bsa_make_overrides    = true;
+
+            profile.base_per_file_settings_.pack = false;
+
+            auto fnv_lod_textures         = PerFileSettings::make_base(game);
+            fnv_lod_textures.pattern      = Pattern(u8".*lod(_[nsdgpe])?\\.dds", Pattern::Type::Regex);
+            fnv_lod_textures.tex_optimize = OptimizeType::None;
+
+            auto fnv_ui_textures         = PerFileSettings::make_base(game);
+            fnv_ui_textures.pattern      = Pattern(u8"textures/interface/.*\\.dds", Pattern::Type::Regex);
+            fnv_ui_textures.tex_optimize = OptimizeType::None;
+
+            auto fnv_dds         = PerFileSettings::make_base(game);
+            fnv_dds.pattern      = Pattern(u8".*\\.dds", Pattern::Type::Regex);
+            fnv_dds.tex.compress = true;
+            fnv_dds.tex.resize   = btu::tex::Dimension{2048, 2048};
```

Comment:
Why? Is there a reason to resize by default?

Reply:
See above reply, basically anything over 2k doesn’t make sense for the game.